packages/mini/test/mini.test.mjs

```diff
@@ -5,42 +5,57 @@ import '@strudel.cycles/core/euclid.mjs';
 describe('mini', () => {
   const minV = (v) => mini(v)._firstCycleValues;
   const minS = (v) => mini(v)._showFirstCycle;
+
   it('supports single elements', () => {
     assert.deepStrictEqual(minV('a'), ['a']);
   });
+
   it('supports rest', () => {
     assert.deepStrictEqual(minV('~'), []);
   });
+
   it('supports cat', () => {
     assert.deepStrictEqual(minS('a b'), ['a: 0 - 1/2', 'b: 1/2 - 1']);
     assert.deepStrictEqual(minS('a b c'), ['a: 0 - 1/3', 'b: 1/3 - 2/3', 'c: 2/3 - 1']);
   });
+
   it('supports slowcat', () => {
     assert.deepStrictEqual(minV('<a b>'), ['a']);
   });
+
   it('supports division', () => {
     assert.deepStrictEqual(minS('a/2'), ['a: 0 - 2']);
     assert.deepStrictEqual(minS('[c3 d3]/2'), ['c3: 0 - 1']);
   });
+
   it('supports multiplication', () => {
     assert.deepStrictEqual(minS('c3*2'), ['c3: 0 - 1/2', 'c3: 1/2 - 1']);
     assert.deepStrictEqual(minV('[c3 d3]*2'), ['c3', 'd3', 'c3', 'd3']);
   });
-  it('supports brackets', () => {
+
+  it('supports sub-cycle', () => {
     assert.deepStrictEqual(minS('c3 [d3 e3]'), ['c3: 0 - 1/2', 'd3: 1/2 - 3/4', 'e3: 3/4 - 1']);
+    assert.deepStrictEqual(minS('c3 . d3 e3'), ['c3: 0 - 1/2', 'd3: 1/2 - 3/4', 'e3: 3/4 - 1']);
     assert.deepStrictEqual(minS('c3 [d3 [e3 f3]]'), ['c3: 0 - 1/2', 'd3: 1/2 - 3/4', 'e3: 3/4 - 7/8', 'f3: 7/8 - 1']);
+    assert.deepStrictEqual(minS('c3 . d3 . e3 f3'), ['c3: 0 - 1/2', 'd3: 1/2 - 3/4', 'e3: 3/4 - 7/8', 'f3: 7/8 - 1']);
```

Comment:
unfortunately, this is not how it works in tidal.. it should have the same result as `[c3] [d3] [e3 f3]`

Reply:
Yes, I'm aware. Converting this to draft for now. Tomorrow I will tackle the grammar again.